### DISCUSSION TITLE
Expand time grid

### DISCUSSION THREAD

**Discussion by Lion1302** [nhn/tui.calendar#1254] [Q&A] [answered]
Hello,
Is there any way to expand time grid for daily mode to have each 15 min interval. Now it has one hour.
![image](https://user-images.githubusercontent.com/70651345/184452358-fcfabb9c-9396-45aa-8616-d00f3fbf8d9e.png)

**Comment by adhrinae** [comment, marked as answer]
No. But Another library like fullcalendar [supports it](https://fullcalendar.io/docs/slotDuration).
